### PR TITLE
Migrate Awsome books to ES6.

### PR DESCRIPTION
**Pull request by NelsonAraujo93**
In this pull request i followed the next steps:

- Set up the linters.
- Install dependencies Luxon.
- Create Index.js, styles.css and index.html files based on Asome Project from last week.
- Create a module Folder and create addSection.js, book.js, bookList.js, contactSection.js and listSection.js.
- Modify files to export and import from index.js.
- Modify index.html to module type.
- Modify package.json add type module.
- Check linters.
- Create ReadMe file.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1 +1,245 @@
...
+
+> Describe your project in 1 or 2 sentences.
+
+**[Awsome Books]** This project is a ES6 project build with JS, CSS and HTML to test our learnings in modular websites building.
+
+## 🛠 Built With <a name="built-with"></a>
+
+### Tech Stack <a name="tech-stack"></a>
+
+> Describe the tech stack and include only the relevant sections that apply to your project.
```
Comment (GzimAsani) [inline README.md]: - [x] Remove description from every header on readme file

File: README.md [hunk 2]
```diff
@@ -1 +1,245 @@
...
+
+```sh
+  cd my-folder
+  git clone https://github.com/NelsonAraujo93/awsomeBooksES6
+```
+--->
+
+### Install
+
+Install this project with:
```
Comment (GzimAsani) [inline README.md]: - [x] Be more explanatory here, explain how your project can be run

File: README.md [hunk 3]
```diff
@@ -1 +1,245 @@
...
+
+```sh
+  bin/rails test test/models/article_test.rb
+```
+--->
+
+### Deployment
+
+You can deploy this project using:
+
```
Comment (GzimAsani) [inline README.md]: - [x] Since you do not have tests please remove these sections

**Review by GzimAsani — CHANGES_REQUESTED** [review]
### Hello @NelsonAraujo93                       :wave:   
 Good job so far! There are some issues that you still need to work on to go to the next project but you are almost there!
 
 ## Highlights
 * Good job adding date from luxon :+1: 

 ## Required Changes 

- [ ] Check the comments below :arrow_double_down: 



 ## Optional suggestions
 _Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._



Cheers and Happy coding! clapclapclap
 
 Feel free to leave any questions or comments in the PR thread if something is not 100% clear. Please, remember to tag me in your question so I can receive the notification.
 
 **_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**
 
 _As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Review by Mihndim2020 — APPROVED** [review]
# STATUS: APPROVED!!!  🎉 🎉 🎉 :champagne: :champagne: :champagne:
Hi @NelsonAraujo93, 

### Good job !!!
### To Highlight! :clap:  :green_circle: 

:heavy_check_mark:  Good commit messages
:heavy_check_mark: Descriptive Readme file

Your project is complete! Great work on making the changes requested by the previous reviewer. There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 🎉 🎉 :champagne: :champagne: :champagne:

Congratulations-Memes-FEATURED

## Optional suggestions

_Check the comments under the review._

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._

### N/A

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear or you can get to me on slack  using this handle @mihndim.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**